### PR TITLE
Added my name!

### PR DESCRIPTION
**Pull request by c-s-ale**
I added my name to the README.md as instructed!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,8 @@ For our purposes, we're going to think of it as a way to sync changes we made to
   6. After that, there are a number of interesting things you can do - and you can check those out in the video, or through the documentation!
   
 **You're all done!** :tada:
+
+
+## Names Here:
```
Comment (chris-alexiuk) [inline README.md:34]: I'd prefer this to say: "## Your name here..."

```suggestion
## Your name here:
```

**Review by chris-alexiuk — CHANGES_REQUESTED** [review]
Hey, I'd love you to fix this typo!

File: README.md [hunk 2]
```diff
@@ -29,3 +29,8 @@ For our purposes, we're going to think of it as a way to sync changes we made to
   6. After that, there are a number of interesting things you can do - and you can check those out in the video, or through the documentation!
   
 **You're all done!** :tada:
+
```
Comment (chris-alexiuk) [inline README.md:32]: Hey, please remove this line of whitespace, I don't like it here!

Comment (c-s-ale) [reply README.md:32]: Okay!

Comment (c-s-ale) [inline README.md:32]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -29,3 +29,7 @@ For our purposes, we're going to think of it as a way to sync changes we made to
...
+## Your name here:
+
+Chris :tada:
```
Comment (chris-alexiuk) [inline README.md:35]:
```suggestion
Chris
```

**Review by chris-alexiuk — APPROVED** [review]
This looks better, thanks!

**Pull request merged**